### DISCUSSION TITLE
No Samba Access To Shares

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#30] [Bug Reports]
Edited.

I have set up a new installation of rAudio on a Pi3B, and access to the smb shares was refused until I restarted smb.
This has to be done after every reboot.

**Comment by R101G** [comment]
No problem since entering _systemctl enable smb_ command, so moved to fixed category.

**Reply by jelinj8** [reply]
Hello, it might be good idea to enable this by enabling SMB in GUI or at least give a hint about that, I've found this by a chance after resigning to SMB usage. Anyway this helped :-)

**Reply by R101G** [reply]
@jelinj8 

Are you saying shares is still not working when you enable it in features? If so please report it as a bug. I assumed it had been fixed.

**Reply by jelinj8** [reply]
Yes, smb service wasn't enabled by enabling smb in the UI (tested on Pi3b+, Pi2B). I've uderstood this thread as enabling it with systemctl is a required step, if it isn't added to the GUI change (the same goes for disabling it when switched off).
Until enabling and starting (or reboot), the rAudio isn't accessible by SMB at all.

**Reply by R101G** [reply]
OK, I jumped the gun and have moved it back to bug reports for @rern attention.
